test/controllers/sessions_controller_test.rb

```diff
@@ -0,0 +1,32 @@
+require "test_helper"
+
+describe SessionsController do
+  let(:user1) { users(:user1) }
+
+  it "should got to login page" do
+    get login_path
+    must_respond_with :success
+  end
+
+  describe "if a user exists" do
```

Comment:
Also test if a new user logs in successfully the number of users goes up and if an existing user logs in the number remains constant.